### ISSUE TITLE
Energy consumption/ coal consumption

### ISSUE THREAD

**Issue by WillyWinkel** [doomeer/factorio#37]
For the furnaces I would like to see how much energy I need in form of coal or electricity, depending on the furnace.
Just something like, you need 4 nuclear reactors to produce 500 yellow science packs/s.
Including uranium at this point would be awesome.
Amazing project btw!

**Comment by breathermachine** [comment]
Yes to this! Now that I'm considering upgrading all of my assembly machine 3s to use four productivity 3 modules, all energized by at least 3 beacons with speed 3s. Some metric with solar panels/accumulators would also be nice.

**Comment by doomeer** [comment]
That's a nice suggestion. Adding this information to all nodes of the tree would probably be too much information, but I could add a summary of required energy production.
